ResearchKit/Localized/ar.lproj/ResearchKit.strings

```diff
@@ -141,6 +141,7 @@
 "PASSCODE_TEXTFIELD_INVALID_INPUT_MESSAGE" = "فقط الأرقام مسموح بها.";
 "PASSCODE_TEXTFIELD_ACCESSIBILITY_LABEL" = "مؤشر تقدم إدخال رمز الدخول";
 "PASSCODE_TEXTFIELD_ACCESSIBILTIY_VALUE" = "تم إدخال %1$@ من %2$@ من الأرقام";
+"PASSCODE_FORGOT" = "Forgot Passcode?";
```

Comment:
Don't need to add this to every non-English language. `ORKLocalizedString()` macro handles it for you.